schoolopy/main.py

```diff
@@ -146,6 +146,26 @@ def get_user(self, user_id):
         """
         return User(self._get('users/%s' % user_id))
 
+    def create_user(self, user_id, user):
+        """
+        Create a new user.
+
+        :param user_id: ID of user you wish to create.
+        :param user: User object containing necessary fields.
+        :return: User object obtained from API.
+        """
+        return User(self._post('users/%s' % user_id, user.json))
+
+    def update_user(self, user_id, user):
+        """
+        Update an existing user.
+
+        :param user_id: ID of user you wish to update.
+        :param user: User object containing necessary fields.
+        :return: User object obtained from API.
+        """
+        return User(self._post('users/%s?update_existing=1' % user_id, user.json))
+
```

Comment:
The other issue here is that this isn't all the options. 

There are also:
* bulk create
* bulk update
* list inactive
* view inactive
* reactivate
* delete
* bulk delete
* A few other you don't have to worry about

Reply:
The list is here, in case you didn't see it.
https://developers.schoology.com/api-documentation/rest-api-v1/user